utils.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+
+import subprocess
+
+
+def pip_install(dependancy):
+    try:
+        subprocess.check_call(["pip", "install", dependancy])
+    except subprocess.CalledProcessError:
+        print("Failed to install '{dependancy}'")
+
+
+def execute_bash(command):
+    try:
+        return subprocess.check_output(command, shell=True, text=True)
```

Comment:
`run` is the [recommended](https://docs.python.org/3/library/subprocess.html#using-the-subprocess-module) method to use.